### ISSUE TITLE
training_logits = tf.identity(training_decoder_output.rnn_output, 'logits') 这里说没有rnn_output属性

### ISSUE THREAD

**Issue by shj0511** [NELSONZHAO/zhihu#27]
是版本问题吗，我的是tensorflow1.10
